TypeScript.YAML-tmLanguage

```diff
@@ -1952,6 +1952,7 @@ repository:
       '0': { name: punctuation.definition.template-expression.end.ts }
     patterns:
       - include: '#expression'
+    contentName: meta.embedded.block.ts
```

Comment:
This should actually probably be `meta.embedded.line` now that I think about it more